cns/ipamclient/api.go

```diff
@@ -0,0 +1,73 @@
+package ipamclient
```

Comment:
Those request and response structures are not exposed to public. That's why I duplicated it.